dev/TabView/TabView.cpp

```diff
@@ -139,7 +140,7 @@ void TabView::OnApplyTemplate()
 
     if (SharedHelpers::IsThemeShadowAvailable())
     {
-        if (!SharedHelpers::Is21H1OrHigher())
+        if (!SharedHelpers::Is21H1OrHigher() && !XamlControlsResources::IsUsingControlsResourcesVersion2())
```

Comment:
This whole change is a bit weird... we are collapsing the shadow caster by default when using the V2 style, but when using the V2 style we will never enter this block, so it doesn't look like we ever make it visible. So then, why does it exist?

Also I don't think I like the idea of our code changing behavior based on the XCR's style version api. why did you need to do this?

Reply:
@StephenLPeters you're absolutely right, I'm confused why I did it this way.
fixed now

Reply:
@StephenLPeters you're absolutely right, I'm also confused as to why I did it this way. It's clearly incorrect.
Fixed now :)